script/FullDeployer.s.sol

```diff
@@ -16,33 +16,34 @@ import {
 
 import {ISafe} from "../src/common/interfaces/IGuardian.sol";
 
-import {HubRequestManager} from "../src/vaults/HubRequestManager.sol";
+import {BatchRequestManager} from "../src/vaults/BatchRequestManager.sol";
 
 import "forge-std/Script.sol";
 
 struct FullReport {
     HubReport hub;
-    HubRequestManager hubRequestManager;
+    BatchRequestManager batchRequestManager;
 }
 
 contract FullActionBatcher is ExtendedHubActionBatcher, ExtendedSpokeActionBatcher, AdaptersActionBatcher {
     function engageFull(FullReport memory report) public onlyDeployer {
         // TODO: should be re-organized
 
         // Rely Root
-        report.hubRequestManager.rely(address(report.hub.common.root));
+        report.batchRequestManager.rely(address(report.hub.common.root));
 
         // Rely others
-        report.hubRequestManager.rely(address(report.hub.hub));
-        report.hubRequestManager.rely(address(report.hub.hubHelpers));
+        report.batchRequestManager.rely(address(report.hub.hub));
+        report.batchRequestManager.rely(address(report.hub.hubHelpers));
 
         // File methods
-        report.hubRequestManager.file("hub", address(report.hub.hub));
+        report.batchRequestManager.file("hub", address(report.hub.hub));
+        report.batchRequestManager.file("gateway", address(report.hub.common.gateway));
```

Comment:
To avoid migrating this, maybe we can do `hub.gateway()` in the few places we need the gateway (?)

Reply:
Let's consider in followup.